### PR TITLE
doc: move dependencies bump batching in general principles

### PR DESCRIPTION
**Pull request by bougue-pe**
Details for editoast are moved close to the code, in `osrd/editoast/README.md` (see https://github.com/osrd-project/osrd/pull/6762).

Follow-up on https://github.com/osrd-project/osrd-website/pull/154#pullrequestreview-1865252374

### CODE REVIEW

**Review by flomonster — APPROVED** [review]

File: content/docs/guides/contribute/contribute-code/general-principles.en.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ description: "Please read this first!"
 - Avoid non well-known abbreviations.
 - **Control and consistency over 3rd party code reuse**: Only add a dependency if it is absolutely necessary.
 - Every dependency we add decreases our autonomy and consistency.
+- Nous essayons de limiter à un petit nombre les PRs de mise à jour des dépendances chaque semaine
+dans chaque composant, donc regrouper les montées de version dans une même PR est une bonne option
+(reportez-vous au `README.md` de chaque composant).
 - **Don't reinvent every wheel**: as a counter to the previous point, don't reinvent everything at all costs.
```
Comment (multun) [inline content/docs/guides/contribute/contribute-code/general-principles.en.md:19]: frenglish

File: content/docs/guides/contribute/contribute-code/general-principles.fr.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ description: "À lire en premier !"
 - Évitez les abréviations peu connues.
 - **Contrôle et cohérence de la réutilisation du code de tiers** : une dépendance est ajoutée seulement si elle est absolument nécessaire.
 - Chaque dépendance ajoutée diminue notre autonomie et notre cohérence.
+- We try to keep PRs bumping dependencies to a low number each week in each component, so grouping
+dependency bumps in a batch PR is a valid option (see component's `README.md`).
 - **Ne pas réinventer la roue** : en opposition au point précédent, ne réinventez pas tout à tout prix.
```
Comment (multun) [inline content/docs/guides/contribute/contribute-code/general-principles.fr.md:18]: frenglish

**Review by multun — CHANGES_REQUESTED** [review]

File: content/docs/guides/contribute/contribute-code/general-principles.en.md (hunk 1 above)
Comment (bougue-pe) [reply content/docs/guides/contribute/contribute-code/general-principles.en.md]: Thanks ! Directly amended into https://github.com/osrd-project/osrd-website/pull/178/commits/6b9188e01c759e254bc6bb98d89414deca05c1df

File: content/docs/guides/contribute/contribute-code/general-principles.fr.md (hunk 2 above)
Comment (bougue-pe) [reply content/docs/guides/contribute/contribute-code/general-principles.fr.md]: Amended into https://github.com/osrd-project/osrd-website/pull/178/commits/6b9188e01c759e254bc6bb98d89414deca05c1df

**Review by multun — APPROVED** [review]

**Pull request merged**